ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/TxControl.java

```diff
@@ -161,20 +161,20 @@ public static boolean isReadonlyOptimisation()
     }
 
     public static final String getXANodeName()
-	{
-		return xaNodeName;
-	}
+    {
```

Comment:
We do not normally change the whitespace because it makes diffs with previous versions more difficult. You have done this in a few places in this PR